exercises/change/change.spec.js

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
All of these tests are present in the pull request.  The pull request actually includes one additional, but trivial test.   The canonical tests return a -1 value for an error condition.  The pull request tests throw an error object, which is the same practice in the Java language tests.   Please advise if returning a -1 value is preferred.